tutorials/pipeline/3_pipeline_dict_with_services_basic.py

```diff
@@ -31,18 +31,18 @@
 
 # %% [markdown]
 """
-When Pipeline is created using `from_dict` method,
+When Pipeline is created using Pydantic's `model_validate` method,
```

Comment:
Since Pipeline no longer has the `from_script` method (and there's no difference between using `__init__` or `model_validate`), I think these tutorials should be merged into one that just goes over Pipeline fields and mentions that it can be initialized from a dictionary.

Reply:
These two tutorials both only had `from_dict`. I think the basic version exists for rookie developers, because they may get caught up reading Pipeline's signature. Sure, expreienced devs will glance at it and keep all that in mind, but newbies may have difficulty processing that. For now I made the merged tutorial in the basic tutorial.